containers/daap-recreate-data-product/src/var/task/recreate_data_product.py

```diff
@@ -0,0 +1,69 @@
+import os
+
+import boto3
+
+
+def handler(event, context):
+    data_product_to_recreate = event.get("data_product", "")
+    raw_data_bucket = os.environ.get("RAW_DATA_BUCKET", "")
+    curated_data_bucket = os.environ.get("CURATED_DATA_BUCKET", "")
+    athena_load_lambda = os.environ.get("ATHENA_LOAD_LAMBDA", "")
+
+    # Check data product has associated data
```

Comment:
This can made as module and called in by passing the boto3,  so it can be tested easily